Library/Fetchers/Fetcher.swift

```diff
@@ -40,10 +40,10 @@ class Fetcher {
         }
     }
 
-    func computePath(for path: String, with parameters: [String: String]? = nil) -> String {
+    func computePath(for path: String, with parameters: [String: AnyHashable]? = nil) -> String {
         var parametersString = ""
         if let parameters = parameters {
-            parametersString = parameters.map { name, value in "\(name.urlEncoded)=\(value.urlEncoded)" }.joined(separator: "&")
+            parametersString = parameters.map { name, value in "\(name)=\(value)" }.joined(separator: "&")
```

Comment:
No, I don't think so. Unless Networking exposed a method to convert a dictionary into URL form-encoded strings that also accent non-string values, which it doesn't, AFAIK. 💭